packages/ingest/src/SftpProviderClient.js

```diff
@@ -144,7 +144,10 @@ class SftpProviderClient {
    * @returns {Promise.<{ s3uri: string, etag: string }>} an object containing
    *    the S3 URI and ETag of the destination file
    */
-  async sync(remotePath, bucket, key) {
+  async sync(params) {
```

Comment:
Making the signature of this (non inherited class) match ProviderClient.   Given time I might put in a seperate PR to typescript this.